src/sdk/lsf-sdk.js

```diff
@@ -184,7 +184,7 @@ export class LSFWrapper {
     try {
       const LSF = await resolveLabelStudio();
 
-      this.globalLSF = window.LabelStudio === LSF;
+      window.LabelStudio === LSF;
```

Comment:
not sure what you are trying to achieve here

Reply:
It was an old idea leftover from a different take on this whole thing, I had changed everything to use loose references and was removing any strictly unused references like this one. Honestly its probably not doing anything given how this evolved.

Reply:
Unused global var?

Reply:
search all the codebases and there is no reference to `globalLSF`. Its just another variable we had set.

Reply:
Just saw this now, its supposed to have been entirely removed not just the assignement 🤣